Library/Homebrew/cask/audit.rb

```diff
@@ -270,6 +271,13 @@ def bad_osdn_url?
       bad_url_format?(/osd/, [%r{\Ahttps?://([^/]+.)?dl\.osdn\.jp/}])
     end
 
+    def check_url_verification
+      return unless cask.url.verified
+      return if cask.url.to_s.start_with?(cask.url.verified)
```

Comment:
The protocol (`http[s]://`) shouldn’t be required for verification.

Reply:
I use regular expression to remove `http[s]://` prefix.